[Bug report] Crash when parsing command-line arguments

[The newly added fuzzer](https://github.com/googleapis/gnostic/pull/171) finds a crash in gnostic.go.

Here are 5 test cases from the fuzzer:
**1:**
```
^@^N


-
```

**2:**
```
t



-
```
**3:**
`:&<86>^@^@      -       ÿ       ÿ       ^@`

**4:**
`^P^K-^M¿`

**5:**
`:       -`


They all follow the same stack trace:
```
panic: runtime error: index out of range [1] with length 1

goroutine 17 [running, locked to thread]:
github.com/googleapis/gnostic/lib.(*Gnostic).readOptions(0x10c000247a40, 0x5, 0x0)
        github.com/googleapis/gnostic/lib/gnostic.go:348 +0x10f1
github.com/googleapis/gnostic/lib.(*Gnostic).Main(0x10c000247a40, 0x0, 0x0)
        github.com/googleapis/gnostic/lib/gnostic.go:598 +0x1b7
github.com/googleapis/gnostic/fuzz.Fuzz(0x602000025070, 0xe, 0xe, 0x18)
        github.com/googleapis/gnostic/fuzz/fuzz.go:29 +0x16a

```

--- a/lib/gnostic.go
+++ b/lib/gnostic.go
@@ -382,7 +382,7 @@ func (g *Gnostic) readOptions() error {
 			g.timePlugins = true
 		} else if arg == "--no-surface" {
 			g.excludeSurface = true
-		} else if arg[0] == '-' && arg[1] == '-' {
+		} else if len(arg) > 2 && arg[0] == '-' && arg[1] == '-' {
 			// try letting the option specify a plugin with no output files (or unwanted output files)
 			// this is useful for calling plugins like linters that only return messages
 			p := &pluginCall{Name: arg[2:len(arg)], Invocation: "!"}

